### DISCUSSION TITLE
Cyfrin Updraft Progress Error

### DISCUSSION THREAD

**Discussion by MihaiHng** [Cyfrin/foundry-full-course-cu#2829] [General]
I just received an error while watching lessons on the platform, now all my progress is gone. I am back to 0%. Is there any issue with the platform? Has anyone else experienced something similar?
The error was something like: "There was an error! You are not authorized for this course". I am halfway through Advanced Foundry.
However, I can can continue the course, just my progress was reset to 0%.

**Comment by EngrPips** [comment]
Good this was fixed.
